testsuite/tests/ui/test_reset_password.py

```diff
@@ -0,0 +1,39 @@
+"""Test for reset password"""
```

Comment:
Imho it can be added as an extension to the current test, no need to create a separate test for this.